templates/layout/base.html

```diff
@@ -2,14 +2,26 @@
 <html class="no-js" lang="{{ locale_name }}">
     <head>
         <title>{{ head.title }}</title>
+
+        <link rel="dns-prefetch preconnect" href="//cdn8.bigcommerce.com" crossorigin>
+        <link rel="dns-prefetch preconnect" href="//cdn7.bigcommerce.com" crossorigin>
+        <link rel="dns-prefetch preconnect" href="//fonts.googleapis.com" crossorigin>
```

Comment:
Also all of this meta needs to be seeded from bcapp so we are in full control if we need to change it in the future without making any updates to the theme.

Reply:
Yep, I will update the cdn preconnect to come from the context. However, the use of Google Fonts is a theme choice, so it seems appropriate to put it in the theme.